src/main/features/response/form/models/statement-of-means/employers.ts

```diff
@@ -28,33 +28,11 @@ export class Employers implements Serializable<Employers> {
     return new Array(rows).fill(EmployerRow.empty())
   }
 
-  deserialize (input?: any): Employers {
-    if (input) {
-      this.rows = this.deserializeRows(input.rows)
-    }
-
-    return this
-  }
-
-  appendRow () {
-    if (this.canAddMoreRows()) {
-      this.rows.push(EmployerRow.empty())
-    }
-  }
-
-  removeExcessRows () {
-    this.rows = this.rows.filter(item => !!item.jobTitle && !!item.employerName)
-
-    if (this.rows.length === 0) {
-      this.appendRow()
-    }
+  getMaxNumberOfRows (): number {
+    return MAX_NUMBER_OF_JOBS
   }
 
-  canAddMoreRows () {
-    return this.rows.length < MAX_NUMBER_OF_JOBS
-  }
-
-  private deserializeRows (rows: any): EmployerRow[] {
+  deserializeRows (rows: any): EmployerRow[] {
```

Comment:
I would remove this to base class as well. Deserialisation logic will be almost the same for all cases.